mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -2751,7 +2751,12 @@ fn electrum_connect(
     );
 
     let connect_loop = select_func(connect_loop.boxed(), shutdown_rx.compat());
-    spawn(connect_loop.map(|_| ()));
+
+    // We can use `spawn` safely since this future will be aborted once `ElectrumConnection` is dropped.
+    unsafe {
```

Comment:
As per this diff https://github.com/KomodoPlatform/atomicDEX-API/compare/stop-spawned-futures...KomodoPlatform:atomicDEX-API:spawn-pub-crate?expand=1, please use `spawn_abortable` here.

Reply:
I decided to make `spawn` and `spawn_local` public and safe.
`ElectrumClient` refactoring was justified, since it is important for us to immediately free all resources on `stop` RPC, although I spent several hours on this.

But in some cases `spawn` and `spawn_local` use is justified, for example, here:
https://github.com/KomodoPlatform/atomicDEX-API/blob/522edd984f77a22b0e0969b33460b0ac0b1048ad/mm2src/mm2_main/src/rpc.rs#L334-L366